src/Components/Inputs/Input.tsx

```diff
@@ -173,6 +182,19 @@ export const Input = forwardRef<InputRef, InputProps>(
 
     const iconElement = icon && <Icon glyph={icon} className="cf-input-icon" />
 
+    const clearClasses = classnames('cf-input-clear-btn', {
+      large: size === ComponentSize.Large,
+    })
+
+    const clearElement = correctlyTypedValue && onClear && (
+      <DismissButton
+        style={clearBtnStyle}
```

Comment:
Do we need to expose this style prop? I think we could just use color = {ComponentColor.default}. I don't see where we would need to change the color or style of the dismiss button within the input but if you can think of concrete future usecases for it, would love to hear it

Reply:
As for styling, I think the default color set is the subtlety we are looking for rather than the primary color that comes out of box with the dismiss button component.

Reply:
are you saying that the color should. be changed from what it currently is?

Reply:
Two things:
1.  I think the Default Color Dismiss button might be more suitable than the Primary Color Dismiss Button. 
2. I'm trying to understand the rationale for style={clearBtnStyle}. It seems like its just passing in undefined.
 In what scenarios would we want to pass in custom style to this dismiss button?